metrics_gather.sh

```diff
@@ -435,12 +435,46 @@ rav1e)
   SIZE=$(stat -c %s $BASENAME.ivf)
   ENC_EXT='.ivf'
   ;;
-svt-av1)
-  export LD_LIBRARY_PATH=$(dirname "$SVTAV1")
-  # svt-av1 has a max intra period of 255
-  $($TIMER $SVTAV1 -i $FILE --preset 0 --lp 1 -q $x -o $BASENAME.yuv -b $BASENAME.ivf -w $WIDTH -h $HEIGHT --intra-period 255 $EXTRA_OPTIONS > $BASENAME-enc.out 2>&1)
-  $YUV2YUV4MPEG $BASENAME -an1 -ad1 -w$WIDTH -h$HEIGHT > /dev/null
-  rm $BASENAME.yuv
+svt-av1 | svt-av1-ra | svt-av1-ra-crf | svt-av1-ra-vbr | svt-av1-ra-vbr-2p | svt-av1-ld-cbr | svt-av1-ra-cq )
+  case $CODEC in
+    # 1-pass CQ, CTC Style Preset
+    svt-av1-ra)
```

Comment:
should we call this svt-av1-ra-cq as it is a cqp only configuration ? also this is specific to the AOM-CTC, do we have a way to specify that ?

Reply:
We already have 1-pass CQP at RA preset, ie. `1-pass CQP RA` at L460, 
https://github.com/xiph/rd_tool/pull/134/files#diff-fad8b2dd3d8d836ab6cddc32eca6846446def6317a738eb0b6f59aadbd8a2723R460


Are you suggesting to merge this with that or have one single preset for RA CQ? 
PS: from users perspective, it will be shown as CTC one,
a)     `"svt-av1-ra": "AV1 (SVT-AV1) 1-Pass CTC Random Access (RA)",`
b)    ` "svt-av1-ra-cq": "AV1 (SVT-AV1) 1-Pass CQP Random Access (RA)",`
https://github.com/xiph/awcy/pull/331/files#diff-095d1a39248780fdbfb1571cfc43ead5a1c93d3b935e90e9d24ab6e3743ecba3R578

Reply:
Having it as AOM CTC works :)

Reply:
thanks, I updated the frontend to reflect that in the PR in the main awcy. 


[also updated presets and QPs here as per discussion]